ansible/almalinux-8-primary/playbook.yml

```diff
@@ -173,6 +173,11 @@
           - ruby
           - ruby-devel
           - rubygem-bundler
+    - name: Set pg_config path for bundler
+      become_user: vagrant
+      shell: |
+        bundle config build.pg \
+          --with-pg-config=/usr/pgsql-{{ postgresql_version.series }}/bin/pg_config
```

Comment:
Could you also add `ansible_ssh_pipelining: true` like other `shell` tasks?
I don't remember the detail but `shell` and `become_user` may fail without it.